src/_components/shared/page_navigation.erb

```diff
@@ -0,0 +1,14 @@
+<page-navigation>
+  <% if @previous.present? %>
```

Comment:
Just asking... We don't need test on `@previous.resource.relative_url` and `@previous.label` presence ?

Reply:
`resource` is a Bridgetown class that always have a `relative_url` and `label` also always exist as a menu entry name.